pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -309,7 +309,7 @@ protected BrokerResponse handleRequest(long requestId, String query, @Nullable S
       JsonNode request, @Nullable RequesterIdentity requesterIdentity, RequestContext requestContext,
       @Nullable HttpHeaders httpHeaders)
       throws Exception {
-    LOGGER.debug("SQL query for request {}: {}", requestId, query);
+    LOGGER.info("SQL query for request {}: {}", requestId, query);
```

Comment:
This will add a lot of log lines for high-qps use-cases.

Reply:
This would just double the lines, given we already emit a info log for each query in `QueryLogger.log`, which is executed at the end of the request.

Reply:
I'm removing it given it is clear this is not a consensuated decision.